packages/fxa-auth-server/lib/sentry.js

```diff
@@ -73,7 +73,7 @@ function filterSentryEvent(event, hint) {
     }
   }
   if (event.tags && event.tags.url) {
-    event.tags.url = event.request.url.replace(TOKENREGEX, FILTERED);
+    event.tags.url = event.tags.url.replace(TOKENREGEX, FILTERED);
```

Comment:
I noticed during copy/paste that I clearly messed up in the auth-server here. 😞

Reply:
Will this be causing any auth-server errors right now?

Reply:
I don't think so, because the request always has a url on it. And I don't think we frequently attach tags with a url.